### ISSUE TITLE
Disable setupClipboard, ignoring paste events

### ISSUE THREAD

**Issue by pknowles** [bgrins/filereader.js#21]
It doesn't look like there's an interface to stop listening for paste events on `<body>` for example. Have I missed something?

**Comment by bgrins** [comment]
You are right - if you call setupClipboard right now there isn't a way to unbind it.  In fact none of the setup calls allow that.  I'd accept a patch to introduce that behavior.  I guess the setup calls could return an object including a destroy() function, or they could check for FileReaderJS.enabled before processing anything (and the page could flip that).
